src/objects.js

```diff
@@ -1,41 +1,55 @@
 const createPerson = (name, age) => {
-  // your code here
+  const myPerson = {
+    name: name,
+    age: age
+  }
+  return myPerson;
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  return object.hasOwnProperty(property);
```

Comment:
JavaScript does not protect the property name hasOwnProperty; an object that has a property with this name may return incorrect results.
The recommended way to overcome this problem is to instead use Object.hasOwn().

Object.hasOwn method takes in two parameters, the object and the property and return true if the object has the property in your situation this would be:
```
  return Object.hasOwn(object, property);
`````